app/src/main/java/com/ryan/heroestopbuilds/MainActivity.java

```diff
@@ -257,9 +257,10 @@ public String prettyPrinter(ArrayList<String> popularSkills) {
         String smSpacing = String.format("%" + 1 + "s", "");
         int weGotAChromie = 0;
         //Check first skills, this means we got a Chromie
-        if(popularSkills.contains("CompoundingAether") || popularSkills.contains("DeepBreath")
-                || popularSkills.contains("TimewalkersPursuit") || popularSkills.contains("PeerIntoTheFuture")) {
-            weGotAChromie = 1;
+        for(String s : popularSkills) {
+            if (s.matches("(?i)(CompoundingAether|DeepBreath|TimewalkersPursuit|PeerIntoTheFuture)")) {
+                weGotAChromie = 1;
```

Comment:
break; 
After this line, if anything matches there is no need to keep looping. Should save cycles.